handbook/engineering/distribution/managed/creation_process.md

```diff
@@ -60,4 +60,18 @@ Creating a new managed instance involves following the steps below.
 			}
 		],
 	```
+1. Contact #ce and ask they generate / provide the relevant license key for the customer's instance, then set it in the site config.
 
+## Giving the customer access
+
+To provide the customer access to the instance:
+
+1. Work with #ce to ask the customer:
+   a. If their instance should be protected by SSO only, or SSO + IP-restricted access. If the latter, what IP addresses / CIDR ranges should be allowed (e.g. the source IPs of their corporate VPN, usually their infrastructure or IT teams can provide this).
+   b. Who should be the recipient of the initial site admin account creation link? This will let them configure the admin password, which they will need to store somewhere securely, and is used for them to set up SSO as well as for them to get access if at any point SSO is not woring. They can create more password or SSO-based admin accounts later as desired.
```

Comment:
```suggestion
   b. Who should be the recipient of the initial site admin account creation link? This will let them configure the admin password, which they will need to store somewhere securely, and is used for them to set up SSO as well as for them to get access if at any point SSO is not working. They can create more password or SSO-based admin accounts later as desired.
```